console/console-init/ui/src/modules/msg-and-iot/dailogs/components/IoTReview/IoTReview.test.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import { IoTReview } from "./IoTReview";
+
+describe("IoT Review", () => {
```

Comment:
use unit test standard as i explain
use some expect

Reply:
expect is not required.